### ISSUE TITLE
wrong input

### ISSUE THREAD

**Issue by Gogumee** [phoenix104104/fast_blind_video_consistency#2]
Hi!

I think your model's input is different from your paper. 

[Code]
### model input        
    inputs = torch.cat((frame_p2, frame_o1, frame_i2, frame_o1), dim=1)

should be 
    inputs = torch.cat((frame_p2, frame_i1, frame_i2, frame_o1), dim=1)

is it right?

Thank you for your good research :)

**Comment by phoenix104104** [comment]
Hi, the order of o1, i1, and i2 doesn't matter if it's consistency during training and testing.

**Comment by musicrainie** [comment]
> Hi!
> 
> I think your model's input is different from your paper.
> 
> [Code]
> 
> ### model input
> ```
> inputs = torch.cat((frame_p2, frame_o1, frame_i2, frame_o1), dim=1)
> ```
> should be
> inputs = torch.cat((frame_p2, frame_i1, frame_i2, frame_o1), dim=1)
> 
> is it right?
> 
> Thank you for your good research :)

According to the test_pretrained.py and the provided results.zip, inputs to the model should be (frame_p2, frame_o1, frame_i2, frame_o1), which surely seems not to comply with the paper.
